content/python/concepts/time-module/sleep.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.sleep()'
+Description: 'The function suspends execution of current thread for a given time.'
+Subject: 'Computer Science'
+Tags: 
+ -'Functions'
+ -'Time'
+CatalogContent:
+ -'learn-python'
+ - 'concepts/time-module'
+---
+
+The python time module function .sleep() suspends execution of current thread for a given period of time(in seconds).
+
+### **Syntax** 
+
+```time.sleep(duration)``` 
+Duration is the time(in seconds) the execution will be suspended/delayed.
+
+This will suspend execution for 23.4 seconds.
+This function is part of the python time module which handles time related tasks.
```

Comment:
```suggestion

The `duration` parameter is given in seconds, and suspends/delays execution of the thread.
```

The last sentence wasn't really necessary since this term entry would be nested under the existing [`time` module concept entry](https://www.codecademy.com/resources/docs/python/time-module).

> This will suspend execution for 23.4 seconds.

Where/how did we arrive at this figure? This information seems to fit better in an actual example, as opposed to the syntax.